nodes/nomos-node/config.yaml

```diff
@@ -7,14 +7,15 @@ consensus:
   fountain_settings: null
   overlay_settings:
     nodes: [[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1]]
+    leader_super_majority_threshold: 1
```

Comment:
This threshold is derived for the tree overlay, but atm node is configured to use FlatOverlay. This will eventually change, env variables will cover both cases.

Reply:
Aaah, ok. We should change the overlay then.